pythoncalculator/multiply.py

```diff
@@ -0,0 +1,2 @@
+def multiply(x,y);
+   return(x+y)
```

Comment:
I think you mixed up addition and multiplication

```suggestion
   return(x*y)
```